.ci/jenkins/Jenkinsfile.setup-branch

```
@@ -62,6 +62,8 @@ pipeline {
                 script {
                     maven.mvnVersionsSet(getMavenCommand(getRepoName()).withProperty('full'), getKogitoVersion(), true)
                     maven.mvnSetVersionProperty(getMavenCommand(getRepoName()), 'version.org.kie', getDroolsVersion())
+                    def path = "${getRepoName()}/quarkus/extensions/kogito-quarkus-workflow-extension-common/kogito-quarkus-workflow-common-deployment/pom.xml"
+                    sh "sed -i -r 's|<(data-index-ephemeral\.image)>(.*):.*|<\1>\2:${getMajorMinorKogitoVersion()}</\1>|' $path"
```

did you try this in Jenkins ? the `\` could cause issues :)

I guess it should be:
```suggestion
                    sh "sed -i -r 's|<(data-index-ephemeral\\.image)>(.*):.*|<\\1>\\2:${getMajorMinorKogitoVersion()}</\\1>|' $path"
```